.github/workflows/notification.yml

```diff
@@ -40,7 +40,7 @@ jobs:
           role-to-assume: ${{ secrets.GU_RIFF_RAFF_ROLE_ARN }}
 
       - name: Compile, test and package
-        run: sbt "project notification" "compile" "test" "debian:packageBin"
+        run: sbt "project common" "compile" "test" "project commonscheduledynamodb" "compile" "test" "project notification" "compile" "test" "debian:packageBin"
```

Comment:
is `commonscheduledynamodb` a shared app?

Reply:
Yes, `notification` depends on both `common` and `commonscheduledynamodb` - https://github.com/guardian/mobile-n10n/blob/main/build.sbt#L163